ui-grid/ui-grid.d.ts

```diff
@@ -3766,6 +3766,14 @@ declare module uiGrid {
          * You may specify one of the sortingAlgorithms found in the rowSorter service.
          */
         sortCellFiltered?: boolean;
+        /**
+		* (optional) An array of sort directions, specifying the order that they should cycle through as 
```

Comment:
In fact we have to restrict the usable values to ASC DESC and null.
How can i do that ? 
I think i should not create a new type because the original ui-grid is using uiGridConstants values.

I will correct spacing.

Thx.